packages/sdk/src/model/ModelClient.ts

```diff
@@ -389,6 +392,35 @@ export class ModelClient extends APIResource {
     }
   }
 
+  /* -------------------------------------------------------------------------
+   * List Model Runs
+   * -----------------------------------------------------------------------*/
+
+  async listModelRunsQuery({
+    modelName,
+    view,
+    pageSize,
+    page,
+    orderBy,
+    filter,
+  }: ListModelRunsRequest) {
+    try {
+      const queryString = getQueryString({
+        baseURL: `${modelName}/runs?view=${view}`,
+        pageSize,
+        page,
+        filter,
+        orderBy,
```

Comment:
@orangecoloured getQueryString has exposed param view, we can use it instead of directly put it in string